plugins/api-docs/package.json

```diff
@@ -30,7 +30,7 @@
     "clean": "backstage-cli clean"
   },
   "dependencies": {
-    "@asyncapi/react-component": "^0.23.0",
+    "@asyncapi/react-component": "1.0.0-next.21",
```

Comment:
Btw, do you really want to use a fixed version here?

Reply:
updated